cumulusci/core/keychain/BaseProjectKeychain.py

```diff
@@ -269,3 +270,8 @@ def list_services(self):
         services = list(self.services.keys())
         services.sort()
         return services
+
+    @property
+    def project_cache_dir(self):
+        "Helper function to get the project_cache_dir from the project_config"
+        return self.project_config.universal_config_obj.cumulusci_config_dir
```

Comment:
9cb8b1b0

Reply:
ab7fe3e874

Reply:
On our infinite todo list it might be nice to add some kind of tests for CCI which would emulate things that would break when it is used in MetaCI and MetaDeploy, because I don't have those use cases internalized at all. In other words: the invariants they expect should be in the CCI test suite so that running the suite would give some confidence that they aren't broken.